modules/le_log/le_log.cpp

```diff
@@ -0,0 +1,94 @@
+#include "le_log.h"
+#include "le_core/le_core.h"
+
+#include <atomic>
+#include <mutex>
+#include <unordered_map>
+#include <cstdarg>
+
+struct le_log_module_o {
+	std::string     name      = "DEFAULT";
+	std::atomic_int log_level = 1;
+};
+
+static le_log_module_o *le_log_module_default() {
+	static auto module = new le_log_module_o();
```

Comment:
this won't be the same after hot-reload, as hot-reloading means invalidating and re-initializing all static memory.

Reply:
I'll try a similar approach to the one use for the textures in le_renderer